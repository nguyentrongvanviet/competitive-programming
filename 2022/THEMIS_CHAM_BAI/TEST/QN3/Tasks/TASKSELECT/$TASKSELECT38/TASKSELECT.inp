10
2 751181431
1 95428815
1 60661253
8 266268991
6 543520704
4 577187805
8 57993792
7 479618947
2 153156342
8 1000000000
