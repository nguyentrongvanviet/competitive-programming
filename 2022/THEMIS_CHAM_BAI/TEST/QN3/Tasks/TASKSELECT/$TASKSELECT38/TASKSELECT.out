375590715
1
1
9143837
516601732
297630164
366250727
860208029
76578170
794293046
